examples/knowledge-base/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "build": "next build",
-    "build:prod": "yarn generate-public-files && next build",
+    "build:prod": "yarn ts-node generate-public-files && next build",
     "start": "next dev",
```

Comment:
As a developer I find it a bit annoying that the changed this from the default Next.js commands. What do you think about changing this back to `dev` so we get `yarn dev` instead og `yarn start`, which I'm used to being what we've called `start:prod`?